include/sharg/detail/format_tdl.hpp

```diff
@@ -286,13 +287,13 @@ class format_tdl : format_base
         }
         for (size_t i{1}; i < executable_name.size(); ++i)
         {
-            auto name = "subcommand_" + std::to_string(i);
-            info.params.push_back(tdl::Node{
-                .name = name,
-                .value = tdl::StringValue(executable_name[i]),
-            });
-            info.cliMapping.emplace_back("", name);
+            parameters = {tdl::Node{
+                .name = executable_name[executable_name.size() - i],
+                .tags = {"basecommand"},
+                .value = parameters,
```

Comment:
The value is again the parameters that is just filled?

Reply:
good question!
I changed slightly what TDL is expecting.
Imagine we have a call like `raptor build --input somefile.fa --kmer 5`, than `--input` and `--kmer` are subparameters of `build`. This is now better reflected in TDL.
But I don't know if someone is calling `raptor ...`, or `raptor build` or something like `git remote add ....`
So I fill `parameters` with `--input` and `--kmer` and than later nest it into a `build` section. (or if there are more subcommands, multiple iterations)

The `basecommand` is a special tag for TDL/CWL expressing that the name has to be part of the command line.